docs/units-and-global-variables.rst

```diff
@@ -182,6 +182,12 @@ Mathematical and Cryptographic Functions
 
     For further details, read `example usage <https://ethereum.stackexchange.com/q/1777/222>`_.
 
+.. warning::
+
+    If you use ``ecrecover``, be aware that a valid signature can be turned into a different valid signature without requiring
+    knowledge of the corresponding private key. This is usually not a problem unless you require signatures to be unique or
+    use them to identify items.
+
```

Comment:
Is there a link where this could point to for more details? After reading this I still have no idea what the problem actually is.

Reply:
@leonardoalt No there isn't, which was my initial problem many months ago, it seems to be a problem, but few know much about it. Maybe the Zeppelin link will help, so I'll add that.

Reply:
late comment here...  the "high s" problem (and why the `ecrecover` precompile behavior was left unchanged when high s became invalid for transactions) is discussed in the Homestead EIP  http://eips.ethereum.org/EIPS/eip-2